### PR TITLE
Print pin code and long discriminator on QR screen

### PR DESCRIPTION
**Pull request by cecille**
This is very helpful info, and it's already part of the QR code
string anyway.

#### Problem
Setup PIN code is hard to find. It's part of the QR code, but that's hard to parse as a human.

#### Change overview
M5 all cluster app only - Prints setup pin code and long discriminator when you go to the QR code screen.

#### Testing
Manual - went to QR code screen, saw prints.

### CODE REVIEW

File: examples/all-clusters-app/esp32/main/main.cpp [hunk 1]
```diff
@@ -669,6 +669,16 @@ extern "C" void app_main()
                    [=]() {
                        ESP_LOGI(TAG, "Opening QR code screen");
                        ESP_LOGI(TAG, "QR CODE Text: '%s'", qrCodeText.c_str());
+                       uint16_t discriminator;
+                       if (ConfigurationMgr().GetSetupDiscriminator(discriminator) == CHIP_NO_ERROR)
+                       {
+                           ESP_LOGI(TAG, "Setup discriminator: %u (0x%x)", discriminator, discriminator);
```
Comment (dhrishi) [inline examples/all-clusters-app/esp32/main/main.cpp:675]: Hi @cecille This is helpful. Just a suggestion: Can you add the print statements in the function `createSetupPayload` where the discriminator and the setup pin code are already populated.

**Review by dhrishi — CHANGES_REQUESTED** [review]

File: examples/all-clusters-app/esp32/main/main.cpp (hunk 1 above)
Comment (cecille) [reply examples/all-clusters-app/esp32/main/main.cpp:675]: Sure - I'll add them there too. I want them on the QR code screen as well so I don't have to go back in the logs when I'm looking for them.

**Review by woody-apple — APPROVED** [review]

**Review by dhrishi — APPROVED** [review]

**Review by jmartinez-silabs — APPROVED** [review]

**Review by andy31415 — APPROVED** [review]
